packages/protocol/contracts/dapis/aggregators/MeanMedianHybrid.sol

```diff
@@ -0,0 +1,98 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.2;
+
+import "./Mean.sol";
+import "./Median.sol";
+import "./Aggregator.sol";
+
+/// @title Mean-median hybrid aggregation contract
+/// @notice Mean is accurate but not robust. Median is inaccurate but robust.
+/// Mean-median hybrid uses mean under normal circumstances for accuracy, but
+/// falls back to median if there is a large enough discrepancy between mean
+/// and median. Since the calculation of mean and its validation takes O(n)
+/// time, it is more scalable than bare median under normal circumstances
+/// (where mean will be similar to median).
+/// @dev Even though the types are defined as `int256` for consistency, this
+/// method is more suitable for unsigned values. For example, consider the
+/// values [-100, -4, 1, 1, 1, 1, 100]. The mean of these values is 0 and the
+/// median is 1. Here, even though the mean is very close to the median, upper
+/// and lower tolerances will be zero, which means the aggregation will fall
+/// back to median. Therefore, consider using Median.sol instead for values
+/// with potentially alternating signs.
```

Comment:
It seems to me this is an edge case when mean == 0? If so, I would just say this explicitly.

Reply:
It's not only when mean == 0 but it becomes more prominent when mean gets closer to 0

Reply:
I don't think so. Obviously it depends on the data, but if you're dealing with small data or data that averages to ~0, then small changes are important, so I think % difference would work in these cases as well. The only real exception is 0. That's why I thought it should be more explicit in the docstring.

Reply:
E.g. consider PNK ;) @ $0.1501, a 1% changes is only fractions of a penny, but it's still significant since the price is so low to begin with.

Reply:
nvm, i agree with you on second thought :) it matters when the number representation is fewer than the max bits stored by the `int`

Reply:
This works well when all values are positive yet small. The problem is when the mean is close to 0 and the values have mixed signs. So it wouldn't be a problem with price data.